### PR TITLE
Add simple styles to example website

### PR DESCRIPTION
**Pull request by crishpeen**

### CODE REVIEW

**Comment by mbohal** [comment]
Please do resolve the conflicts.

File: src/languages/cs_CZ.yml [hunk 1]
```diff
@@ -1,2 +1,3 @@
-Languages: Jazyky
-Contact: Kontakt
+Example: Ukázka
+Home: Úvod
+Contact: Kontakt
```
Comment (adamkudrna) [inline src/languages/cs_CZ.yml]: No newline at end of file

File: web/css/main.css [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+
+.error {
+    color: #C74451;
+}
+
+.img-responsive {
+    display: block;
+    max-width: 100%;
+    height: auto;
+}
```
Comment (adamkudrna) [inline web/css/main.css]: No newline at end of file

File: src/templates/_layouts/base.html.twig [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+        <div class="container">
+            <h1><a href="{{ path('index') }}">{{ 'Example'|trans }}</a></h1>
+            <nav>
+                <ul>
+                    <li{% if page == 'index' %} class="active"{% endif %}><a href="{{ path('index') }}">{{ 'Home'|trans }}</a></li>
+                    <li{% if page == 'contact' %} class="active"{% endif %}><a href="{{ path('contact') }}">{{ 'Contact'|trans }}</a></li>
+                </ul>
+                {% block languages %}
+                    <ul class="languages">
+                        <li{% if locale == 'cs_CZ' %} class="active"{% endif %}><a href ="{{ path(page, {}, 'cs_CZ') }}">Cs</a></li>
```
Comment (adamkudrna) [inline src/templates/_layouts/base.html.twig]: Please capitalize language codes (CS, EN) or use full language names (Czech, English).

File: src/templates/cs_CZ/product.html.twig [hunk 4]
```diff
@@ -1,17 +1,19 @@
...
 {% set prod = '' %}
 {% for product in data.products if product.cs.slug == url_params.product_slug %}
     {% set prod = product %}
 {% endfor %}
 
 {% block languages %}
-    <a href ="{{ path(page, { 'product_slug': prod.cs.slug }, 'cs_CZ') }}">Cs</a>
-    <a href ="{{ path(page, { 'product_slug': prod.en.slug }, 'en_US') }}">En</a>
+    <ul class="languages">
+        <li{% if locale == 'cs_CZ' %} class="active"{% endif %}><a href ="{{ path(page, { 'product_slug': prod.cs.slug }, 'cs_CZ') }}">Cs</a></li>
```
Comment (adamkudrna) [inline src/templates/cs_CZ/product.html.twig]: Here goes the same with the languages.

File: src/templates/en_US/product.html.twig [hunk 5]
```diff
@@ -1,17 +1,19 @@
...
 {% set prod = '' %}
 {% for product in data.products if product.en.slug == url_params.product_slug %}
     {% set prod = product %}
 {% endfor %}
 
 {% block languages %}
-    <a href ="{{ path(page, { 'product_slug': prod.cs.slug }, 'cs_CZ') }}">Cs</a>
-    <a href ="{{ path(page, { 'product_slug': prod.en.slug }, 'en_US') }}">En</a>
+    <ul class="languages">
+        <li{% if locale == 'cs_CZ' %} class="active"{% endif %}><a href ="{{ path(page, { 'product_slug': prod.cs.slug }, 'cs_CZ') }}">Cs</a></li>
```
Comment (adamkudrna) [inline src/templates/en_US/product.html.twig]: Here goes the same with the languages.

**Review by adamkudrna — CHANGES_REQUESTED** [review]

**Review by mbohal — APPROVED** [review]

**Review by adamkudrna — APPROVED** [review]

**Pull request merged**